app/consts.go

```diff
@@ -28,23 +44,44 @@ var (
 
 func ModuleBasics() sdkmodule.BasicManager {
 	// Set prefixes
-	accountPubKeyPrefix := AccountAddressPrefix + "pub"
-	validatorAddressPrefix := AccountAddressPrefix + "valoper"
-	validatorPubKeyPrefix := AccountAddressPrefix + "valoperpub"
-	consNodeAddressPrefix := AccountAddressPrefix + "valcons"
-	consNodePubKeyPrefix := AccountAddressPrefix + "valconspub"
+	accountPubKeyPrefix := Bech32PrefixAccPub
+	validatorAddressPrefix := Bech32PrefixValAddr
+	validatorPubKeyPrefix := Bech32PrefixValPub
+	consNodeAddressPrefix := Bech32PrefixConsAddr
+	consNodePubKeyPrefix := Bech32PrefixConsPub
 
 	// Set and seal config
 	config := sdk.GetConfig()
 	config.SetBech32PrefixForAccount(AccountAddressPrefix, accountPubKeyPrefix)
 	config.SetBech32PrefixForValidator(validatorAddressPrefix, validatorPubKeyPrefix)
 	config.SetBech32PrefixForConsensusNode(consNodeAddressPrefix, consNodePubKeyPrefix)
 
-	moduleBasics := sdkmodule.BasicManager{}
-	depinject.Inject(
-		depinject.Configs(AppConfig()),
-		&moduleBasics,
+	moduleBasics := sdkmodule.NewBasicManager(
+		auth.AppModuleBasic{},
+		genutil.AppModuleBasic{},
+		bankModule{},
+		capability.AppModuleBasic{},
+		stakingModule{},
+		mint.AppModuleBasic{},
+		distributionModule{},
+		govModule{},
+		params.AppModuleBasic{},
+		crisisModule{},
+		slashingModule{},
+		authzmodule.AppModuleBasic{},
+		feegrantmodule.AppModuleBasic{},
+		ibc.AppModuleBasic{},
+		evidence.AppModuleBasic{},
+		transfer.AppModuleBasic{},
+		vesting.AppModuleBasic{},
+		blobmodule.AppModuleBasic{},
+		bsmodule.AppModuleBasic{},
+		upgrade.AppModuleBasic{},
 	)
+	// depinject.Inject(
```

Comment:
Why did you do here?
This is Cosmos SDK v0.50's way

Reply:
That way, the decoder does not recognize the type of Msg.
This occurs not only with blobs, but also with banks, etc.
```unable to resolve type URL /sunrise.blob.v1.MsgPayForBlobs: tx parse error```

Reply:
hmm

This way for workaround is fine but we should follow the v0.50's way as far as possible I think